x-pack/plugin/ent-search/src/main/java/org/elasticsearch/xpack/application/connector/syncjob/action/ListConnectorSyncJobsAction.java

```diff
@@ -125,10 +150,25 @@ public XContentBuilder toXContent(XContentBuilder builder, Params params) throws
                 builder.field(PAGE_PARAMS_FIELD.getPreferredName(), pageParams);
                 builder.field(CONNECTOR_ID_FIELD.getPreferredName(), connectorId);
                 builder.field(ConnectorSyncJob.STATUS_FIELD.getPreferredName(), connectorSyncStatus);
+                builder.field(ConnectorSyncJob.JOB_TYPE_FIELD.getPreferredName(), connectorSyncJobTypeList);
             }
             builder.endObject();
             return builder;
         }
+
+        private List<ConnectorSyncJobType> stringToEnumList(List<String> jobTypeList) {
+            if (jobTypeList == null) {
+                return null;
```

Comment:
Should we return an empty list here to avoid NPEs at other places? 

This method and also `enumToStringList` could also live inside `ConnectorSyncJobType` as `fromStringList` and `toStringList`. I guess this could come in handy for others to use, too.

Reply:
>Should we return an empty list here to avoid NPEs at other places?

This is intended, to keep it consistent. If `null` is provided as the param(`List<String>`), it should be `null` after being converted to an enum list.


> This method and also `enumToStringList` could also live inside `ConnectorSyncJobType` as `fromStringList` and `toStringList`.

`ConnectorSyncJobType` is an Enum class, which already has `toString` and `fromString` methods. It may make sense to put it in an util package to make it more general to accept any Enum class. WDYT?

Reply:
> This is intended, to keep it consistent. If null is provided as the param(List<String>), it should be null after being converted to an enum list.
Totally fair 👍 

> ConnectorSyncJobType is an Enum class, which already has toString and fromString methods. It may make sense to put it in an util package to make it more general to accept any Enum class. WDYT?

Sounds good to me :-)